b2b-sample/README.md

```diff
@@ -131,16 +124,23 @@ To create a user for Best Auto Mart with permissions to create an IdP for the **
         "name": "Guardio Insurance - Administrator Application",
         "tag": "Administrator Application"
       }
+    },
+    "ManagementAPIConfig": {
+      "SharedApplicationName": "Guardio-Business-App",
+      "UserStore" : "DEFAULT"
```

Comment:
Moved the configs to an appropriate place as discussed in the [comment](https://github.com/wso2/samples-is/pull/405/files#r1133118476)